### PR TITLE
Jleslie/docker desktop pre req 06 10

### PR DESCRIPTION
**Pull request by jleslie85**
1. Updates and suggestions from Gerry resulting from his testing. Gerry tested the Docker Desktop path. 
2. Also fixed several typos and improved wording choices in several places.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 # The NuoDB Operator
 
-[![Build Status](https://travis-ci.org/nuodb/nuodb-operator.svg?branch=master)](https://travis-ci.org/nuodb/nuodb-operator)
```
Comment (jleslie85) [inline README.md]: I put this back. Looks like a github common link to include. 
Is there anyway to clear the error, run a correct travis build I suspect? 
Otherwise, I think we just have to live with the error message on the page (or remove the link if eng. advises it's ok).

File: deploy/crds/nuodb_v2alpha1_nuodb_cr.yaml [hunk 2]
```diff
@@ -5,18 +5,18 @@ metadata:
 spec:
   dbAvailability: manual
   storageMode: ephemeral
-  insightsEnabled: true
-  adminCount: 1
+  insightsEnabled: false
+  adminCount: 3
   adminStorageSize: 5G
-  adminStorageClass: local-disk
+  adminStorageClass: hostpath
```
Comment (rshaull) [inline deploy/crds/nuodb_v2alpha1_nuodb_cr.yaml:11]: Why this change?

Comment (jleslie85) [reply deploy/crds/nuodb_v2alpha1_nuodb_cr.yaml:11]: This is to default to something more useful (in this case, Docker Desktop storage class 'hostpath') rather than "local-disk" (which is our storage class for manually creating node storage) which we really do not want users to use unless absolutely necessary.
